src/workerd/io/worker-entrypoint.c++

```diff
@@ -729,6 +729,11 @@ kj::Promise<bool> WorkerEntrypoint::test() {
       [](IoContext& context, kj::Own<IoContext::IncomingRequest> request) -> kj::Promise<bool> {
     TRACE_EVENT("workerd", "WorkerEntrypoint::test() waitForFinished()");
     auto result = co_await request->finishScheduled();
+
+    if (result == IoContext_IncomingRequest::FinishScheduledResult::ABORTED) {
+      co_await context.onAbort();
```

Comment:
I would add a comment:

> Force the abort exception to be rethrown from the test handler. This is important because the abort exception is not otherwise logged as an uncaught exception elsewhere, so if we don't rethrow it, the test log won't actually say why the test failed (although the test will still fail, by returning `false` below).